src/python/dxpy/utils/job_log_client.py

```diff
@@ -43,8 +43,36 @@ class DXJobLogStreamingException(Exception):
 class DXJobLogStreamClient:
     def __init__(
         self, job_id, input_params=None, msg_output_format="{job} {level} {msg}",
-        msg_callback=None, print_job_info=True
+        msg_callback=None, print_job_info=True, exit_on_failed=True
     ):
+        """Initialize job log client.
+
+        :param job_id: dxid for a job (e.g. 'job-123' style)
+        :type job_id: str
+        :param input_params: blob with connection parameters, should have keys
+        ``numRecentMessages`` (int) (wich may not be more than 1024 * 256 otherwise no logs returned),
+        ``recurseJobs`` (bool) - if True, attempts to traverse subtree
+        ``tail`` (bool) - if True, keep watching job. Should also be set to true to get all logs
+        from a completed job.
+        :type input_params: dict
+        :param msg_output_format: how messages should be printed to console. Ignored if
+        ``msg_callback`` specified
+        :type msg_output_form: str
+        :param print_job_info: if True, prints metadata about job
+        :type print_job_info: bool
+        :param msg_callback: single argument function that accepts a JSON blob with message
+        details. blob looks like
+        ``{"timestamp":1575465039481,"source":"APP","level":"STDOUT","job":"job-123","line":24,"msg":"success WfFragment"}``
+        where ``timestamp`` is milliseconds since epic and ``line`` is a sequence number.
+        :type msg_callback: callable
+        :param exit_on_failed: if True, will raise SystemExit with code of 3 if encountering a
+        failed job (this is the default behavior)
+        :type exit_on_failed: bool
+
+        NOTE: If you change the functionality of this client, please contact downstream users (including Myriad
```

Comment:
added this because there aren't tests and I'm hoping we can get a ping to make sure this still emits logs in the same way - I know it's a bit goofy :-/